### ISSUE TITLE
Change layout direction when language changed

### ISSUE THREAD

**Issue by emadimehrdad** [akexorcist/Localization#27]
Hi, how can i change layout direction when a user changes his app's language

**Comment by JanuszHain** [comment]
if resources put in localized packages doesn't work, put strings in string xmls, different for each language, read and and depending on it change layout direction in code - quick shot, I guess it is doable

**Comment by dimkinware** [comment]
I'm using next code in BaseActivity class
```
    @Override
    public void setContentView(@LayoutRes int layoutResID) {
        super.setContentView(layoutResID);
        View v = findViewById(android.R.id.content);
        if(v!=null) {
            Locale locale = LanguageSetting.getLanguage(this);
            v.setLayoutDirection(TextUtils.getLayoutDirectionFromLocale(locale));
        }
    }
```

**Comment by yosefi1988** [comment]
@dimkinware  TNX

**Comment by Rissmon** [comment]
@dimkinware It worked. Thanks
@akexorcist Thanks for this awsome work.

**Comment by akexorcist** [comment]
@dimkinware Does #10  solution work as well? I just merged it to 1.2.4

**Comment by dimkinware** [comment]
> @dimkinware Does #10 solution work as well? I just merged it to 1.2.4

Looks like yes
